src/test/config/convert/shadow.expected.yaml

```diff
@@ -1,28 +1,23 @@
 general:
   stop_time: 3600
 topology:
-  graphml: |-
-    <?xml version='1.0' encoding='utf-8'?>
-    <graphml xmlns="http://graphml.graphdrawing.org/xmlns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns http://graphml.graphdrawing.org/xmlns/1.0/graphml.xsd">
-      <key attr.name="prefer_direct_paths" attr.type="string" for="graph" id="d5" />
-      <key attr.name="packet_loss" attr.type="double" for="edge" id="d4" />
-      <key attr.name="latency" attr.type="double" for="edge" id="d3" />
-      <key attr.name="bandwidth_up" attr.type="string" for="node" id="d2" />
-      <key attr.name="bandwidth_down" attr.type="string" for="node" id="d1" />
-      <key attr.name="country_code" attr.type="string" for="node" id="d0" />
-      <graph edgedefault="undirected">
-        <data key="d5">True</data>
-        <node id="poi-1">
-          <data key="d0">US</data>
-          <data key="d1">81920 Kibit</data>
-          <data key="d2">81920 Kibit</data>
-        </node>
-        <edge source="poi-1" target="poi-1">
-          <data key="d3">50.0</data>
-          <data key="d4">0.0</data>
-        </edge>
-      </graph>
-    </graphml>
+  gml: |
+    graph [
+      prefer_direct_paths 1
```

Comment:
Does this need `directed 0` like shown in the other GML graphs?

Reply:
The default for gml is undirected, so if the graph is undirected, networkx doesn't include the `directed` key. I'll update the xml to use the `edgedefault="directed"` attribute in this conversion test so that we can make sure networkx sets this correctly.

Reply:
If you look at the other GML graphs in this commit, why do those list `directed 0`?

Reply:
I hand-wrote those graphs and included it just to be explicit, but I can remove those if it's more clear.

Reply:
Ahh, OK, this all makes sense now! I think you can just ignore this comment.